Admin JS tests is broken

### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Lint and tests Github action is failing because feature flag config files have been move to Woocommerce plugin.

https://github.com/woocommerce/woocommerce/runs/5881994933?check_suite_focus=true



### Expected behavior

Lint and tests action should work as expected

### Actual behavior

```
    Cannot find module '../../../../plugins/woocommerce-admin/config/development.json' from '../../../packages/js/js-tests/build/setup-globals.js'

      101 |     });
      102 | });
    > 103 | const config = require('../../../../plugins/woocommerce-admin/config/development.json');
          |                ^
      104 | // Check if test is jsdom or node
      105 | if (global.window) {
      106 |     window.wcAdminFeatures = config && config.features ? config.features : {};

```

### Steps to reproduce

1. `pnpm nx test woocommerce-admin`

### WordPress Environment

na

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

--- a/packages/js/js-tests/src/setup-globals.js
+++ b/packages/js/js-tests/src/setup-globals.js
@@ -107,7 +107,7 @@ wooCommercePackages.forEach( ( lib ) => {
 	} );
 } );
 
-const config = require( '../../../../plugins/woocommerce-admin/config/development.json' );
+const config = require( '../../../../plugins/woocommerce/client/admin/config/development.json' );
 
 // Check if test is jsdom or node
 if ( global.window ) {

